### PR TITLE
Remove mentions of TCP

### PR DESCRIPTION
**Pull request by NoCatt**
Removed the mention of TCP in the prequesits  to avoid confusing smart northstar users

### CODE REVIEW

**Comment by itscynxx** [comment]
We don't need [final checks](https://github.com/NoCatt/NorthstarWiki/blob/removeTCP/docs/hosting-a-server-with-northstar/prerequisites.md#final-checks) anymore do we? It only checks TCP afaik

**Comment by ASpoonPlaysGames** [comment]
image
boooo

File: docs/hosting-a-server-with-northstar/prerequisites.md [hunk 1]
```diff
@@ -1,6 +1,9 @@
 # Prerequisites
 
-**TL;DR:** Port forward `37015` (UDP). If using Northstar v1.12 or older, also port forward `8081` (TCP).
+**TL;DR:** Port forward `37015` (UDP), and no [CGNAT](prerequisites.md#cgnat)
+
+Note that by default your server will be called ``Unnamed Northstar Server``. You can change the name as described in the next page.
+
```
Comment (itscynxx) [inline docs/hosting-a-server-with-northstar/prerequisites.md]: this new line isn't needed

File: docs/hosting-a-server-with-northstar/prerequisites.md [hunk 2]
```diff
@@ -54,18 +56,3 @@ If you accidentally click the deny button, then follow the step to allow it.
...
-
-You can also use a third-party site like [this one](https://www.ipfingerprints.com/portscan.php) to see whether you set up the TCP port forwarding correctly for the authentication port (usually `8081`) by performing a scan while your Northstar server is running.
-
-![Example check](https://raw.githubusercontent.com/R2Northstar/NorthstarWiki/main/docs/images/portforwarding-testing.png)
-
-![Working portforwarding for the TCP port](https://raw.githubusercontent.com/R2Northstar/NorthstarWiki/main/docs/images/portforwarding-working.png)
-
-Testing port forwarding like this only works for the TCP port due to the way UDP and Northstar work.
-
-Note that by default your server is called `Unnamed Northstar Server`. You can change this name as described in the next page.
```
Comment (GeckoEidechse) [inline docs/hosting-a-server-with-northstar/prerequisites.md]: I would keep the _Final Checks_ section with the exception of the of the TCP portscanning part.

Also when removing it, make sure to also remove the linked images so we don't end up with leftover files ^^

**Review by itscynxx — CHANGES_REQUESTED** [review]
small thing

**Review by itscynxx — APPROVED** [review]
Issues I had are resolved, don't see any more obvious mentions of TCP on wiki myself

File: docs/hosting-a-server-with-northstar/basic-listen-server.md [hunk 3]
```diff
@@ -25,8 +25,7 @@ Below are a series of variables and commands you can use for server configuratio
...
 | `ns_auth_allow_insecure`                         | Allows clients to join your server without authenticating with the masterserver, currently required to allow clients to connect directly to your IP, rather than through the server browser | `0`                            |
-| `ns_erase_auth_info`                             | Whether your server should erase authentication information after it is used, this is useful for development but should normally be kept at 1                                               | `1`                            |
-| `ns_player_auth_port`                            | Northstar v1.12 and older only. The port used for the server's local authentication server, this is the TCP port we forwarded earlier                                                       | `8081`                         |
+| `ns_erase_auth_info`                             | Whether your server should erase authentication information after it is used, this is useful for development but should normally be kept at 1                                               | `1`                            |
```
Comment (GeckoEidechse) [inline docs/hosting-a-server-with-northstar/basic-listen-server.md]: Accidental trailing newline introduced here

```suggestion
| `ns_erase_auth_info`                             | Whether your server should erase authentication information after it is used, this is useful for development but should normally be kept at 1                                               | `1`                            |
```

File: docs/hosting-a-server-with-northstar/prerequisites.md [hunk 4]
```diff
@@ -1,6 +1,8 @@
 # Prerequisites
 
-**TL;DR:** Port forward `37015` (UDP). If using Northstar v1.12 or older, also port forward `8081` (TCP).
+**TL;DR:** Port forward `37015` (UDP), and no [CGNAT](prerequisites.md#cgnat)
+
+Note that by default your server will be called ``Unnamed Northstar Server``. You can change the name as described in the next page.
```
Comment (GeckoEidechse) [inline docs/hosting-a-server-with-northstar/prerequisites.md]: I'd drop the second line byline. Keep it short and simple. The TL;DR is kinda intended for someone that hosted servers for other games before. So even CGNAT mention might be overkill

```suggestion
**TL;DR:** Port forward `37015` (UDP), and no [CGNAT](prerequisites.md#cgnat)
```

File: docs/hosting-a-server-with-northstar/troubleshooting.md [hunk 5]
```diff
@@ -44,10 +44,6 @@ If your port can be accessed from your local IP but not from your public IP, the
 
 See [CGNAT](https://r2northstar.gitbook.io/r2northstar-wiki/hosting-a-server-with-northstar/prerequisites#cgnat)
 
-### If server is not reachable using external IP nor using internal IP
```
Comment (GeckoEidechse) [inline docs/hosting-a-server-with-northstar/troubleshooting.md:47]: Why are you removing the section header?

**Review by GeckoEidechse — CHANGES_REQUESTED** [review]

File: docs/hosting-a-server-with-northstar/prerequisites.md (hunk 2 above)
Comment (GeckoEidechse) [reply docs/hosting-a-server-with-northstar/prerequisites.md]: This still unaddressed ^^

Comment (NoCatt) [reply docs/hosting-a-server-with-northstar/prerequisites.md]: > This still unaddressed ^^

I addessed it in the dc, where I said that this is not the page for this, furthermore cyn and I _sorta_ agreed to put it in another page

Comment (GeckoEidechse) [reply docs/hosting-a-server-with-northstar/prerequisites.md]: Didn't notice anything in the discussion on Discord that mentioned it and besides, Discord is not the place to resolve PR reviews, as it cannot be easily viewed again at a later date.

With all that said, removing the section should be separate PR. Just remove the part that refers to the removed TCP port and keep the rest for this PR.

File: docs/hosting-a-server-with-northstar/prerequisites.md [hunk 6]
```diff
@@ -58,14 +57,4 @@ If you accidentally click the deny button, then follow the step to allow it.
...
-
-You can also use a third-party site like [this one](https://www.ipfingerprints.com/portscan.php) to see whether you set up the TCP port forwarding correctly for the authentication port (usually `8081`) by performing a scan while your Northstar server is running.
-
-![Example check](https://raw.githubusercontent.com/R2Northstar/NorthstarWiki/main/docs/images/portforwarding-testing.png)
-
-![Working portforwarding for the TCP port](https://raw.githubusercontent.com/R2Northstar/NorthstarWiki/main/docs/images/portforwarding-working.png)
-
-Testing port forwarding like this only works for the TCP port due to the way UDP and Northstar work.
-
-Note that by default your server is called `Unnamed Northstar Server`. You can change this name as described in the next page.
```
Comment (GeckoEidechse) [inline docs/hosting-a-server-with-northstar/prerequisites.md:71]: The last line is still missing :>

Comment (NoCatt) [reply docs/hosting-a-server-with-northstar/prerequisites.md:71]: literally sooooo unneeded

File: docs/hosting-a-server-with-northstar/prerequisites.md [hunk 7]
```diff
@@ -59,13 +58,4 @@ If you accidentally click the deny button, then follow the step to allow it.
...
+Note that by default your server is called `Unnamed Northstar Server`. You can change this name as described in the next page.
```
Comment (GeckoEidechse) [inline docs/hosting-a-server-with-northstar/prerequisites.md]: Newline for formatting

```suggestion

Note that by default your server is called `Unnamed Northstar Server`. You can change this name as described in the next page.
```

File: docs/hosting-a-server-with-northstar/prerequisites.md [hunk 8]
```diff
@@ -60,12 +59,4 @@ If you accidentally click the deny button, then follow the step to allow it.
...
```
Comment (GeckoEidechse) [inline docs/hosting-a-server-with-northstar/prerequisites.md]: No clue why GitHub still keeps highlighting this as changed. It neither has trailing whitespace nor missing newline so no clue what's causing it

Comment (GeckoEidechse) [reply docs/hosting-a-server-with-northstar/prerequisites.md]: I swear if this is `CRLF` vs `LF` I'm gonna be really annoyed >:(

Comment (GeckoEidechse) [reply docs/hosting-a-server-with-northstar/prerequisites.md]: Oh apparently it was missing a newline before. Wtf?

Comment (NoCatt) [reply docs/hosting-a-server-with-northstar/prerequisites.md]: aaaand that matters how?

Comment (GeckoEidechse) [reply docs/hosting-a-server-with-northstar/prerequisites.md]: Oh no, nevermind. This PR removes the newline. And GitHub vscode extension just ignored that in the diff, ugh. Had to go in manually by checking out branch locally to detect.

Comment (GeckoEidechse) [reply docs/hosting-a-server-with-northstar/prerequisites.md]: > aaaand that matters how?

Affects the diff and `git blame` and so it should be avoided if possible. Not the end of the world but stuff like this makes things messy in the long run. Anyway I fixed it manually now.

**Review by GeckoEidechse — APPROVED** [review]
Requested changes should all be addressed now and apart from the RCON page (which is a separate issue) this PR removes all mentions of `TCP` related things.

Removing TCP mentions is fine as TCP requirement was removed in `v1.12` and we have since version gated to `v1.17` so no version of Northstar currently connected to master server has the TCP requirement (with the exception of some ancient dev servers).
